js/build.js

```diff
@@ -10,9 +10,14 @@ Fliplet.Widget.instance('chart-pie-1-1-0', function(data) {
   if (themeInstance) {
     Object.assign({}, themeInstance.data.values);
 
+    var themeValue = themeInstance.data.values;
+    var widgetValue = getColors(themeInstance.data.widgetInstances);
+
+    themeValues = Object.assign(themeValue, widgetValue);
+
     _.forEach(themeInstance.data.widgetInstances, function(widgetProp) {
-      if (chartId === widgetProp.id) {
-        Object.assign(themeValues, widgetProp.values);
+      if (chartId === widgetProp.id && themeValues) {
```

Comment:
Why is this part necessary? I imagine after `Object.assign()` in the line above, it will always be truthy.

```suggestion
      if (chartId === widgetProp.id) {
```

If not, please give me the full scenario of when `themeValues` would be falsey.